documentation/asciidoc/accessories/display/legacy.adoc

```diff
@@ -1,6 +1,31 @@
 == Legacy Support
 
-NOTE: These instructions are for the original Raspberry Pi 1 Model A and B boards only. You can identify an original board as it is the only model with a 26-pin GPIO header, all other models have the now-standard 40-pin connector.
+[NOTE]
```

Comment:
This is a big NOTE block, should probably be at the end rather than right at the start of the section.

Reply:
OTOH, if this was moved to the _end_ of the Legacy section, people might think that the "are for the original Raspberry Pi 1 Model A and B boards only" disclaimer applies too? :man_shrugging:

Reply:
That was my thinking on it, @lurch. Otherwise it'd likely need a heading to separate it from the old hardware instructions and, well, that sort of negates the idea of a note block.